docs/configuration/CONFIGURATION_SUPPORTED_EXTRACTOR.md

```diff
@@ -40,48 +33,41 @@ For an up-to-date documentation, run `ng help @o3r/components:extractor`
       "@your/o3r-components"
     ]
   }
-},
+}
 ```
 
-__Note:__ This options will not search for the duplicate configurations in libraries.
+__Note:__ This option will not search for the duplicate configurations in libraries.
 
 The __strict mode__ option should be used for production builds, it will throw an error if something not supported by the
-cms is found in the config. You can set it to `false` to allow the generation of metadata including the unknown types to
+CMS is found in the config. You can set it to `false` to allow the generation of metadata including the unknown types to
```

Comment:
if an issue (error + inconstancy)  is found in the config. "supported by the CMS" do not mean much in this context I think